clients/http/utils/request.go

```diff
@@ -233,6 +233,12 @@ func DeleteRequest(ctx context.Context, returnValuePointer interface{}, baseUrl
 	if err != nil {
 		return errors.NewCommonEdgeXWrapper(err)
 	}
+
+	if len(res) == 0 {
+		// no response body is returned from the http request
+		return nil
+	}
```

Comment:
all the http method request may encounter the same issue.
please add the same check in all the functions in this file before the json unmarshal

Reply:
Done